subprojects/introspect/source/jcli/introspect/data.d

```diff
@@ -538,11 +537,19 @@ import std.meta;
 // NOTE:
 // Passing the udas as a sequence works, but trying to get them within the function does not.
 // Apparently, it may work if we mark the function static, but afaik that's a compiler bug.
-ArgFlags foldArgumentFlags(udas...)()
+// Another NOTE:
+// We still need the static here?? wtf?
+static ArgFlags foldArgumentFlags(udas...)()
```

Comment:
Might be falling victim to this amazingly annoying feature of templates: https://dlang.org/spec/template.html#implicit-nesting